tests/main/classic-unencrypted-fakeinstaller/task.yaml

```diff
@@ -0,0 +1,118 @@
+summary: Ensure installer APIs works on unencrypted devices
```

Comment:
I wonder if we should remove this test in favor of `nested/manual/fakeinstaller` - they are similar and the only upside of this test is that it will be a bit quicker but it also does not test booting into the generated image so I'm inclined to say we should just not use it (as the nested one does the same and more).

Reply:
Agreed, I've removed it